Gemfile

```diff
@@ -16,8 +17,6 @@ gem 'coffee-rails', '~> 4.1.0'
 
 # Use jquery as the JavaScript library
 gem 'jquery-rails'
-# Turbolinks makes following links in your web application faster. Read more: https://github.com/rails/turbolinks
-gem 'turbolinks'
```

Comment:
We're also removing Turbolinks, since it's designed for a different type of application than that one you'd make with Angular.